### PR TITLE
Chapter 8 Basic Login

### PR DESCRIPTION
**Pull request by Iskandar19**

### CODE REVIEW

**Comment by Iskandar19** [comment]
Ready

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -14,3 +14,6 @@
 //= require activestorage
 //= require turbolinks
 //= require_tree .
+//
```
Comment (ngocvt-0484) [inline app/assets/javascripts/application.js]: delete it please

File: config/routes.rb [hunk 2]
```diff
@@ -1,9 +1,14 @@
 Rails.application.routes.draw do
+  get 'sessions/new'
```
Comment (ngocvt-0484) [inline config/routes.rb]: remove it because you defined at line 11

**Comment by Iskandar19** [comment]
Ready

**Review by ghost — COMMENTED** [review]
merged

**Pull request merged**